### DISCUSSION TITLE
Auto-login for specific time/date?

### DISCUSSION THREAD

**Discussion by JB-Prof** [ltsp/ltsp#876] [Q&A]
Hello.

We have some classrooms where the users log into the clients as they need, and it works great.

Due to some scheduling changes, we're now facing a situation where we'd like the client computer to auto-login at a specific time & date, but the rest of the week it would allow users to log in with their own accounts.  

For example:
Tuesdays at 9 am - client computer automatically logs in with a predetermined username & password.  The rest of the week it has the usual login prompt and the user manually logs in.

Is there a way to do this please?

Thank you!

**Comment by alkisg** [comment]
The Greek version of "ltsp-manager", which is called "sch-scripts", supports that via an autologin script:

https://gitlab.com/sch-scripts/sch-scripts/-/blob/main/share/sch-scripts/autologin

You can run that via epoptes at any time, or via cron at scheduled times, and it'll automatically login any class/group of students.
Unfortunately at this point there are no plans to make it internationally available, so you'd have to take it as a starting point and adjust its code to suit you...

**Reply by JB-Prof** [reply]
Hello.

I am now trying to implement this on our new LTSP server, and the script runs, but what happens is the client goes into an endless loop.  It switches from the greeter to the text console, some text flashes past, and then the login screen appears for a second, then it switches back to text, and so on endlessly.  I have to reboot the client to make it stop.

I am not sure how to even see the logs on the thin client since I need to reboot it to make it stop.  How can I resolve this please?

Thank you.
